core/functions.php

```diff
@@ -1,47 +1,76 @@
 <?php
 
 if (!function_exists('getDatabaseConnection')) {
-    function getDatabaseConnection($reconnect = FALSE)
+    /**
+     * @param bool $reconnect
+     * @return \PDO
+     */
+    function getDatabaseConnection($reconnect = false)
```

Comment:
This version of Obray requires at least PHP 8.0 so you could append return types to this and the other helper functions you've modified here.

Additionally, we should call out the side-effects of this function (setting the global `$conn` parameter) in the docblock.

Reply:
baafa5b4601a78451f71e509d8878e21fa1d3dc4

Reply:
[9d8a6e1](https://github.com/MaloufSleep/obray/pull/28/commits/9d8a6e12c5529e546bf3adbaff69dfc2ff092fab)
[a9174c9](https://github.com/MaloufSleep/obray/pull/28/commits/a9174c90415f65ae3d54585cc3eccb6634763d6f)